components/src/dynamo/planner/core/base.py

```
@@ -697,6 +750,231 @@ async def _apply_effects(self, effects: PlannerEffects) -> None:
+                    active_decode_blocks_threshold=theta_d_set,
+                    active_prefill_tokens_threshold=theta_p_abs_set,
+                    active_prefill_tokens_threshold_frac=theta_pf_set,
+                )
+                for pod in pods
+            ),
+            return_exceptions=True,
+        )
+
+        failed = [
+            p.metadata.name
+            for p, r in zip(pods, results)
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)